src/components/StyleGuide/Bidirectionality.mdx

```diff
@@ -163,7 +166,7 @@ directionality should be implemented at the application level. The following are
 general guidelines on how to set up directionality but implementation will be up
 to the application and its architecture.
 
-### `dir` HTML attribute
+### \`dir\` HTML attribute
```

Comment:
Ha!  I ran into a problem with this formatting in another story.  I took the easy way out and just removed the backticks.